node/index.js

```diff
@@ -444,6 +444,9 @@ function TChannelConnection(channel, socket, direction, remoteAddr) {
 
     self.handler.on('call.incoming.response', function onCallResponse(res) {
         var op = self.popOutOp(res.id);
+        if (!op) {
+            return;
```

Comment:
Hmm, if you hit this then I'd ask please implement the pre-existing TODO in popOutOp:
```
        // TODO else case. We should warn about an incoming response for an
        // operation we did not send out.  This could be because of a timeout
        // or could be because of a confused / corrupted server.
```